kubedoom.go

```diff
@@ -64,24 +70,37 @@ type Mode interface {
 type podmode struct {
 }
 
+func dontPanicPtr[a any](ret *a, err error) *a {
```

Comment:
Shouldn't this be called `panicOnErrPtr`? My golang is weak but seems like it will panic if there's an error?